src/flamenco/log_collector/fd_log_collector.h

```diff
@@ -8,6 +8,14 @@
 #include "../../ballet/base64/fd_base64.h"
 #include <stdio.h>
 #include <stdarg.h>
+#include <alloca.h>
+
+/* FD_BASE58_ENCODE_32 is a shorthand for fd_base58_encode_32, including
+   defining a temp array.  Useful for printf-like functions. */
+#define FD_BASE58_ENCODE_32(x) (__extension__({            \
```

Comment:
Please move this to fd_flamenco_base.h since macro is reusable everywhere in runtime code